src/app/code/community/FireGento/Pdf/Model/Observer.php

```diff
@@ -46,6 +47,36 @@ public function addInvoiceNotes(Varien_Event_Observer $observer)
     }
 
     /**
+     * Add notes based on shipping country
+     *
+     * @param  Varien_Event_Observer $observer observer object
+     *
+     * @return $this
+     */
+    public function addShippingCountryNotes(Varien_Event_Observer $observer)
+    {
+        $order = $observer->getOrder();
+        $shippingCountryId = $order->getShippingAddress()->getCountryId();
+        $countryNotes = unserialize(Mage::getStoreConfig('sales_pdf/invoice/shipping_country_notes'));
+
+        $note = '';
+        foreach ($countryNotes as $countryNote) {
+            if ($countryNote['country'] == $shippingCountryId) {
+                $note = $countryNote['note'];
+                break;
```

Comment:
it is possible to have more than one line per country. I know, this doesn't make much sense, but I think we shouldn't just break here, instead collect all and return them with \n imploded